docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss

```diff
@@ -1,4 +1,4 @@
-.hb-layout-builder-main-content .views-exposed-form {
+.views-exposed-form { // TODO: ask Merani about Reset button hover state
```

Comment:
@heatherdesigns did you get a chance to ask Merani?

Reply:
I didn't ask because I saw where the styles were there but not displaying until I added `!important`. 🤷